### DISCUSSION TITLE
How to best track a "There was a problem submitting the form." error

### DISCUSSION THREAD

**Discussion by Hannoma** [laravel/nova-issues#4967] [Q&A] [answered]
The create form does not work correctly in one of my nova resources. I am getting a "There was a problem submitting the form." error but there aren't any validation errors highlighted/displayed in the form. Also, no HTTP requests are made (and therefore failing) in the background. Is there a good way to track the issue down?

**Comment by Hannoma** [comment, marked as answer]
I fixed the issue. It was a faulty custom field which was not meant to be displayed on the creation form. Tracked it down by removing fields one by one until it worked. A bit tedious.

**Reply by piljac1** [reply]
Next time, you should take a look at your laravel.log file for more info :)
